python/makeAlpacaEvents.py

```diff
@@ -0,0 +1,146 @@
+import os,sys,getopt
+import numpy,math
+import ROOT
+
+hGeV = 6.58211928*pow(10.,-16)* pow(10.,-9) # no units or it messes up!!
+c_light = 2.99792458e+10
+mres="1.0d0"
+gax="1d-7"
+nev="100"
+
+def AlpacaFormatting(s):
+    s=str(s)
+    if s.find('d')==-1:
+        s = numpy.format_float_scientific(float(s))
+        s= s.replace('e','d')
+        s= s.replace('+','')
+    return s
+
+#determine the path for online usage it can be /eos/../ship/data/.. BUT don't know what we shall do in local usage??
```

Comment:
Maybe use <root://eospublic.cern.ch//eos/....>, which works from anywhere and give an option to use a local file via `-f localfilename.root`?

Reply:
does it work even for ```/afs/cern.ch/...```?
Then, nice!

Reply:
Unfortunately not (but data files should be on EOS anyway).

Note, this only works when opening files with ROOT (however, you can use ROOT to open any file by using a `?raw` suffix to the filename and then it can be used similarly to the usual python `open` function.

Reply:
> Unfortunately not (but data files should be on EOS anyway).
Yes, indeed. The reason of that comment was this.

Reply:
Now, ntuples are saving into eos.

Reply:
Ntuples should probably not be written to EOS directly (imagine use in HTCondor). Reading from EOS is great, but writing should ideally happen to some local scratch space and then the job can take care of copying files to EOS.

Sorry for the confusion!

Reply:
Okay, I am going to fix it as soon as my jobs are done in HTCondor.